refinery/factory_boy/utils.py

```diff
@@ -212,3 +212,362 @@ def create_tool_with_necessary_models(tool_type, user=None):
     if user is not None:
         tool.set_owner(user)
     return tool
+
+
+def create_hg_19_data_set(user=None):
```

Comment:
For these two: `create_hg_19_data_set` & `create_isatab_9909_data_set` maybe call them:`create_mock_*_data_set`